### ISSUE TITLE
Clarification for Customer Feedback

### ISSUE THREAD

**Issue by cagesellchen** [AlyssaRicketts/IDE-IT-Frontend#18]
tldr; I messed up

Hey, 

I wanted to send an apology/clarification email about the lack of customer feedback this quarter. I've been the one on my team doing most of the customer feedback, and this whole time I've been submitting it to the backend team. The original assignments for Proj 2 linked us to the backend team, so I just assumed that since your projects were connected, you were doing the reports together. So, this whole quarter, the backend team has been getting two sets of feedback on their report, and you've gotten none. 

I figured this out tonight because we got an email from Rashmi saying that we hadn't submitted feedback to you, and after some digging around my team realized I was submitting it to the wrong place. How it's taken this long, I have no idea, but I was submitting it to canvas and getting full points, so I never thought anything was wrong.

In any case, it was completely my fault, and I'm really sorry it's taken so long to figure out. It's probably a bit late to give helpful feedback on past assignments, but we'll submit new feedback for Proj 7. If there's anything else I can do to help (or if you do want feedback on past assignments), please let me know!

Again, I'm really sorry!

Best,
Annie Gesellchen
geselc@uw.edu
